pkg/test/framework/components/apps/apps.go

```diff
@@ -65,6 +65,13 @@ type App interface {
 type AppParam struct {
 	Name     string
 	Locality string
+	// ServiceAnnotations specifies the annotations for the deployment.
```

Comment:
`for the deployment` or for the Service?

Reply:
done.